OC4IDS_Data_Feedback_Notebook.ipynb

```diff
@@ -20,7 +20,7 @@
       "version": "3.8.2"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    df = pd.read_csv('https://docs.google.com/spreadsheets/d/1Vo6-Jis-J61PB_33QQx1YKnmQnI7M4rTq6CTMckAFsE/export?format=csv&amp;gid=835339370')
</pre><p>Use descriptive variable names, e.g. <code>indicators</code> rather than <code>df</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>